x-pack/plugins/spaces/public/space_selector/space_selector.tsx

```diff
@@ -106,104 +104,111 @@ export class SpaceSelector extends Component<Props, State> {
     }
 
     return (
-      <EuiPage className="spcSpaceSelector" data-test-subj="kibanaSpaceSelector">
-        <EuiPageBody>
-          <EuiPageHeader className="spcSpaceSelector__heading">
-            <EuiSpacer size="xxl" />
-            <span className="spcSpaceSelector__logo">
-              <EuiIcon size="xxl" type={`logoElastic`} />
-            </span>
-
-            <EuiTitle size="l">
-              <h1 tabIndex={0} ref={this.setHeaderRef}>
-                <FormattedMessage
-                  id="xpack.spaces.spaceSelector.selectSpacesTitle"
-                  defaultMessage="Select your space"
-                />
-              </h1>
-            </EuiTitle>
-            <EuiText size="s" color="subdued">
-              <p>
-                <FormattedMessage
-                  id="xpack.spaces.spaceSelector.changeSpaceAnytimeAvailabilityText"
-                  defaultMessage="You can change your space at anytime"
-                />
-              </p>
-            </EuiText>
-          </EuiPageHeader>
-          <EuiPageContent className="spcSpaceSelector__pageContent">
-            <EuiFlexGroup
-              // @ts-ignore
-              direction="column"
-              alignItems="center"
-              responsive={false}
-            >
-              {this.getSearchField()}
-            </EuiFlexGroup>
-
-            <EuiSpacer size="xl" />
-
-            {this.state.loading && <EuiLoadingSpinner size="xl" />}
-
-            {!this.state.loading && (
-              <SpaceCards spaces={filteredSpaces} serverBasePath={this.props.serverBasePath} />
-            )}
-
-            {!this.state.loading && !this.state.error && filteredSpaces.length === 0 && (
-              <Fragment>
-                <EuiSpacer />
-                <EuiText color="subdued" textAlign="center">
+      <KibanaPageTemplate
+        template="empty"
+        className="spcSpaceSelector"
+        data-test-subj="kibanaSpaceSelector"
+        pageContentBodyProps={{ className: 'spcSpaceSelector__pageContent' }}
+      >
+        <EuiText textAlign="center" size="s">
+          <EuiSpacer size="xxl" />
+          <KibanaSolutionAvatar name="Elastic" size="xl" />
+          <EuiSpacer size="xxl" />
+          <h1
+            // plain `eui` class undos forced focus style on non-EUI components
+            className="eui spcSpaceSelector__pageHeader"
+            tabIndex={0}
```

Comment:
Related to my previous comment, do we need the `tabIndex={0}` here? Is this for accessibility reasons? I thought the practice of using `tabIndex` on non-interactive elements like this was a bit of an anti-pattern.

Reply:
Agreed... but I wasn't going to mess with accessibility "fixes"